lambdas/death-minimisation/src/main/java/uk/gov/di/data/lep/DeathMinimisation.java

```diff
@@ -43,6 +45,10 @@ public String handleRequest(SQSEvent sqsEvent, Context context) {
             var minimisedData = objectMapper
                 .writer(filterProvider)
                 .writeValueAsString(enrichedData);
+
+            var auditData = generateAuditData(minimisedData, enrichedData.txn());
```

Comment:
I'd be half tempted to shift the generation and the auditing into their own method, as `auditMinimsation` or something similar - we don't need the auditData return value for anything else

Reply:
This would apply to the other cases too

Reply:
Yeah I think it was because it was a lot more doable to commonise the auditing rather than the generating I went for this, but can have another look at it

Reply:
Probably not worth spending too much time on this, I think it's very marginal!